components/entities/api/src/test/java/org/phenotips/entities/internal/DefaultPrimaryEntityResolverTest.java

```diff
@@ -119,7 +121,10 @@ public void setUp() throws Exception
 
         this.referenceResolver = this.mocker.getInstance(DocumentReferenceResolver.TYPE_STRING, "current");
 
-        this.component = this.mocker.getComponentUnderTest();
+        this.component = (DefaultPrimaryEntityResolver) spy(this.mocker.getComponentUnderTest());
+
+        doReturn(true).when(this.component).isValidManager(this.familyResolver);
+        doReturn(true).when(this.component).isValidManager(this.patientResolver);
```

Comment:
I'm not a fan of this. You're bypassing an important check in the class.

...However, there's no easy way of not doing this.